integration/meta-typings-as-const/google/protobuf/descriptor.ts

```diff
@@ -167,7 +167,7 @@ export interface FieldDescriptorProto {
    * If true, this is a proto3 "optional". When a proto3 field is optional, it
    * tracks presence regardless of field type.
    *
-   * When proto3_optional is true, this field must be belong to a oneof to
```

Comment:
@NathanBaulch these `google/*/*.ts` files/comments come from comments within the google `*.proto` files that we can't change, so I think we need to revert any `descriptor.ts` changes.

Reply:
Done!